client/src/components/OpportunitiesList.js

```diff
@@ -238,46 +243,19 @@ class OpportunitiesList extends Component {
           <br></br>
         </Divider>
         <br />
+        {filteredOpportunities.length === 0 ? (

```

Comment:
```filteredOpportunities.length === 0``` is same as ```!filteredOpportunities.length``` so try to use the second way which is more convenient. 

Besides, it will be good if you add an additional check to make sure there is opportunities before use start searching like this: ```opportunitiesList && !filteredOpportunities.length```

Reply:
ok! I will change it